### PR TITLE
Allow text search providers to give messages

### PR DESCRIPTION
**Pull request by JacksonKearl**
Allow a text search provider to give a message with additional information regarding a completed search:

image
image

### CODE REVIEW

File: src/vs/workbench/contrib/search/browser/searchView.ts [hunk 1]
```diff
@@ -1452,6 +1457,13 @@ export class SearchView extends ViewPane {
 					type: MessageType.WARNING
 				});
 			}
+			console.log({ completed });
```
Comment (pixieaka) [inline src/vs/workbench/contrib/search/browser/searchView.ts:1460]: Console log statement.

Comment (JacksonKearl) [reply src/vs/workbench/contrib/search/browser/searchView.ts]: Fixed, thanks

**Review by roblourens — COMMENTED** [review]
Should that message show in the yellow popup like that we use when you hit the search limit? Is it a warning?

**Comment by JacksonKearl** [comment]
Not a warning, just info. Nothing wrong with the user deciding they're happy with the search as-is, and we'd also like to use this to say things like "Searching based on XXX index".

**Review by roblourens — APPROVED** [review]

**Review by andrewchoi5 — APPROVED** [review]

**Pull request merged**